src/client/components/Dashboard/Dashboard.jsx

```diff
@@ -17,9 +17,7 @@ export default function Dashboard() {
     const [isCreateSessionModalOpen, setIsCreateSessionModalOpen] = useState(false);
     const [isSessionMoreInfoModalOpen, setIsSessionMoreInfoModalOpen] = useState(false);
     const [selectedSessionId, setSelectedItemId] = useState(null);
-    const sessions = useSelector(state => state.fetchSession);
-    const [cardList, setCardList] = useState(sessions);
-    const [secondCardList, setSecondCardList] = useState(sessions.slice(3, 7));
```

Comment:
This was for displaying different lists for the presentation. Must be changed with filters or another UI design

Reply:
Yeah, as it wasn't being used correctly right now I got rid of it and fixed adding new sessions to the dashboard. We can add a filter in the future to separate the items in the dashboard by sport or something.